collections/Add_A/MS_Add_A_188.xml

```diff
@@ -53,122 +57,180 @@
                      <idno type="SCN">29157</idno>
                   </altIdentifier>
                </msIdentifier>
+               <msContents resp="#VD">
+                  <msItem class="#liturgica" n="1" xml:id="MS_Add_A_188-part2-item1">
+                     <locus>(fols. 1r–167v)</locus>
+                     <title key="work_12559" type="desc">Lectionary with musical cues </title><note>(partly adapted into a <title type="desc" key="work_13355">Missal</title>)</note>
+                     <note>In addition to the epistle and gospel readings of a Lectionary, there are the opening words with superscript musical notation for the introit, gradual, Alleluia versicle or tractus, offertory and sometimes also of the communion antiphon.</note>
+                     <msItem>
+                        <locus>(fols. 1r–136v)</locus>
+                        <title type="desc">Temporale</title>
+                        <rubric>Dominica prima de adventu domini</rubric>
+                        <incipit>Antiphona. Ad te levavi animam. Ad Romanos. Fratres. Scientes quia hora est</incipit>
+                        <note>The Passion narratives with superscript letters: t = Christ, c = Narrator, a = the Jews (cf. K. Young, ‘Observations on the origin of the mediæval Passion-play’, <title>PMLA</title>, 25.2 (1910), 309–54 at 316, cites other Bodleian MSS. with this combination of letters, but not the present MS.): Matthew (fols. 54r–58v), Mark (fols. 60r–63v), Luke (fols. 64v–68r), and John (fols. 69v–72r).</note>
+                        <note>The Exultet given in full, noted (fols. 72v–73v).</note>
+                        <note>With the complete baptismal rite on Holy Saturday (fol. 75v–81v), ending with a short litany. Next to the rubric ‘Hic ponantur cerei ardentes in fontem &amp; insuffletur in eundem fontem ad similitudinem huius figurę’ is a large marginal Greek letter psi: Ψ (fol. 80r).</note>
+                        <note>The volume has been partly adapted into a noted missal by near-contemporary marginal additions from Easter to the 11th Sunday after Pentecost, but coloured initials and musical notation are often lacking (fols. 82v–117v).</note>
```

Comment:
It might be helpful to have a few examples of the texts being used to define this as a missal.